lib/cosmos/tools/cmd_tlm_server/interface_thread.rb

```diff
@@ -148,51 +148,54 @@ def handle_packet(packet)
         if packet.identified?
           begin
             # Preidentifed packet - place it into the current value table
-            identified_packet = System.telemetry.update!(packet.target_name,
-                                                         packet.packet_name,
-                                                         packet.buffer)
+            identified_packet = System.telemetry.packet(packet.target_name, packet.packet_name)
```

Comment:
Where else is update! used? Without all the bang methods how does the current value table get updated?